### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by pcppcp**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People *love* thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+TBD: there'll be a requirement to pass the linter before merging the PR
+
+## License
+By contributing, you agree that your contributions will be licensed under its MIT License.
```
Comment (danielinux) [inline CONTRIBUTING.md]: This says MIT, while COPYING says apache 2.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Contributing to Anyledger wallet
```
Comment (danielinux) [inline CONTRIBUTING.md]: This whole file looks like a copy-paste from another project from briandk. It even has a few parts written in first person. Are you sure we have the right to derivate from that document? What is the license of the original file? Did you ask the original author if it was OK to use this as guidelines in other projects? Are you sure this applies to us? The license seems wrong, and other things seem a bit off...

File: CONTRIBUTING.md (hunk 1 above)
Comment (pcppcp) [reply CONTRIBUTING.md]: MIT is compatible with Apache. Contributions can be done under MIT and relicensed to whatever we want.

File: CONTRIBUTING.md (hunk 2 above)
Comment (pcppcp) [reply CONTRIBUTING.md]: >This whole file looks like a copy-paste from another project from briandk

... which is based on copy-paste of facebook's guidelines.

Comment (danielinux) [reply CONTRIBUTING.md]: still, there are reference in first person, so this look pretty amateuristic

File: CONTRIBUTING.md (hunk 1 above)
Comment (danielinux) [reply CONTRIBUTING.md]: still, it's not the right license

File: CONTRIBUTING.md (hunk 2 above)
Comment (lorepieri8) [reply CONTRIBUTING.md]: I agree, let's reword in third person.

**Comment by lorepieri8** [comment]
@danielinux @pcppcp  
This can be an interesting source of inspiration:
https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md

**Comment by lorepieri8** [comment]
Done in #115.

**Pull request closed without merging**